wurst/_handles/Destructable.wurst

```diff
@@ -69,6 +69,78 @@ public function destructable.setOccluderHeight(real height)
 public function destructable.getName() returns string
 	return GetDestructableName(this)
 
+// Extensions
+/** used to open a destructable gate*/
+public function destructable.gateOpen()
+	if (GetDestructableLife(this) > 0)
+		KillDestructable(this)
+
+/** used to close a destructable gate*/
+public function destructable.gateClose()
```

Comment:
Ditto the above - let's use tupletypes to add some type-safety for gates, elevators, switches.